### PR TITLE
plotly.js images filename

### PR DESCRIPTION
**Pull request by SnowNooDLe**
# Customization

Update based on Jason's feedback.

Instead of using Station, we use Lat & Lng (with 4 dp)

image

### CODE REVIEW

**Review by claudio525 — APPROVED** [review]

File: frontend/src/components/Hazard/SeismicHazard/HazardBranchPlot.js [hunk 1]
```diff
@@ -73,6 +73,9 @@ const HazardBranchPlot = ({ hazardData, im }) => {
         useResizeHandler={true}
         config={{
           displayModeBar: true,
+          toImageButtonOptions: {
+            filename: `Branches_Hazard_Plot_${im}_${station}`,
```
Comment (sungeunbae) [inline frontend/src/components/Hazard/SeismicHazard/HazardBranchPlot.js:77]: Just a question. Is there any chance where `im` or `station` contains underscores?

Comment (SnowNooDLe) [reply frontend/src/components/Hazard/SeismicHazard/HazardBranchPlot.js:77]: I only played station with CCCC so cannot confirm it but IM, yes, pretty much everything but PGA has underscore.

**Review by jasonmotha — APPROVED** [review]
looks good. As claudio mentioned a while back. Station_name is internal to us. So we probably don't need it on the filename but instead maybe lat/lon?

File: frontend/src/components/Hazard/SeismicHazard/HazardBranchPlot.js (hunk 1 above)
Comment (claudio525) [reply frontend/src/components/Hazard/SeismicHazard/HazardBranchPlot.js:77]: I don't think there are any stations that contain underscores, in terms of the IMs with underscores I think just removing the underscore would be fine, i.e. pSA_5.0 becomes pSA5.0
Actually on that, we don't want dots in the name, so for the IMs those should also be changed to p
I.e. this is what I recommend:
`f"Branchs_Hazard_Plot_{im.replace('_', '').replace('.', 'p')}_{station}"`
The above is python code, no idea how to do this in javascript

Also jason's point below is also good, lat  & lon makes more sense

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**